metricbeat/module/jolokia/jmx/jmx.go

```diff
@@ -69,16 +62,19 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 	http.SetMethod("POST")
 	http.SetBody(body)
 
+	log := logp.NewLogger(metricsetName)
+
 	if logp.IsDebug(metricsetName) {
```

Comment:
@andrewkroh Do we have any direct replacment method for `IsDebug`?

Reply:
Nope. Calling `IsDebug` is mostly unnecessary when using the structured logger because the fields are lazily marshaled to a log message. And for the print and printf style methods the string formatting is not performed until after checking if the logging level is enabled. So the only performance hit is in the allocation of the slice that holds the fields.